### ISSUE TITLE
Support for CFNet (CVPR 2019)

### ISSUE THREAD

**Issue by netw0rkf10w** [zhanghang1989/PyTorch-Encoding#269]
Hi Hang,

Congratulations on your awesome work! I am really impressed!

I would like to know if you could share the code and pretrained models for your "Co-Occurrent Features in Semantic Segmentation" paper. (You've made all your implementations available, including the recent ResNeSt, but this CFNet seems to be an exception currently.)

I am looking for a strong semantic segmentation baseline in PyTorch, and according to [this page](https://paperswithcode.com/sota/semantic-segmentation-on-pascal-voc-2012) your CFNet seems to be the best one.

Thank you very much in advance!

**Comment by zhanghang1989** [comment]
Thanks for reminding me. I just recently switch back to PyTorch. I will provide pretrained models for CFNet soon.

Btw, I am also training a stronger model using ResNeSt. Please check back in about 1~2 weeks. Stay tuned.

**Comment by netw0rkf10w** [comment]
@zhanghang1989 That's great! I am looking forward to the updates. Thanks for the prompt reply!

**Comment by soroushmbk** [comment]
Hi

thanks for the great code and papers.
I am interested to work on your "Co-Occurrent Features in Semantic Segmentation" paper. did you publish the code? 

the atten.py file in models directory has similar modules with CFNet. What's the relation between this file and CFNet?

best regards

**Comment by zhanghang1989** [comment]
Yes, `atten.py` is the code for CFNet. Using the hyper-parameters provided in the paper should reproduce the results. Note that the learning rate may be half of the regular settings for some datasets.

**Comment by soroushmbk** [comment]
thank you very much.
